packages/appium/lib/extension/manifest.js

```diff
@@ -0,0 +1,564 @@
+// @ts-check
+
+/**
+ * Module containing {@link Manifest} which handles reading & writing of extension config files.
+ */
+
+import {env, fs, mkdirp} from '@appium/support';
+import _ from 'lodash';
+import path from 'path';
+import YAML from 'yaml';
+import {DRIVER_TYPE, PLUGIN_TYPE} from '../constants';
```

Comment:
we probably need to follow one style here - either we keep spaced after and before object curly brackets or we don't

Reply:
right, so, the reason this _sometimes_ happens is that we don't seem to have anything that automates the spaces around the imports.  I try to remember to do it, but my formatter will sometimes remove them.